### ISSUE TITLE
Finish this library ?

### ISSUE THREAD

**Issue by smlarkin** [pablo-abc/solid-reach#1]
This library has not had a commit for 8 months.

Will it continue to evolve / be feature complete ?

**Comment by pablo-abc** [comment]
Yeah, I paused development on this to focus on v1 of [Felte](https://github.com/pablo-abc/felte). While I would like to finish this one (accesible component libraries are definitely needed), I just haven't managed to juggle the time to develop this. Specially since Felte gets more usage/open issues.

There's other minor things. Reach's API relies heavily on some features of React (such as React.children) that Solid does not provide. This makes the Solid version not have such a nice API. For example [the tooltip component needs to have a function as its child](https://github.com/pablo-abc/solid-reach/blob/main/packages/tooltip/Tooltip.stories.tsx). So I've been also considering if there might be a more "Solid" way to make an accessible component library.

The components available right now were "easy" to migrate since, for the most part, the code could be the same.

---

There's also the fact that [Reach's development has also slowed a bit](https://github.com/reach/reach-ui/commits/develop). I'm guessing due to their focus on Remix. And since I started working on this, other alternatives have appeared on React land such as [Radix](https://www.radix-ui.com) which have an arguably better API and have made me wonder if going for a 1:1 API with Reach would be worth it at all.

---

Finally, there's the fact that you can easily use custom elements with Solid which means that you can use libraries such as [Shoelace](https://shoelace.style) with _waaaay_ more contributors and usage, so there'd be more confidence on it staying up to date with a11y standards.

---

Anyway, since Felte v1 is already released I do want to find some time to either keep working on this or start working on a more "Solid" alternative.

**Comment by smlarkin** [comment]
@pablo-abc thank you for taking the time to thoroughly answer me :)

I would like to work on a component library for Solid that has these attributes...

- Accessibility baked in and done right
- Great primitives used to build complex interactions (Reach / Radix)
- Atomic props that use CSS-in-JS for base stylings 
- An easy way to integrate with UnoCSS for styling and to override defaults / give more options than the Atomic Props
- An optional / not required theme provider for design systems (so uses baked in defaults if a theme prop is not sent)
- A complete library full of plainly styled options (ready for customization) including everything needed for web & mobile PWA (like Ionic)

...The plan was to start where you left off and then start experimenting.  If you have any comments or ideas, that would be helpful.  Cheers!

**Comment by pablo-abc** [comment]
So, ReachUI (and other "primitive only" libraries) generally don't have an API for theming. But that'd be interesting to work on, although it wouldn't be a "port" per se of another library.

I saw [this on Solid's discord](https://github.com/LXSMNSYC/solid-headless). It's a port of HeadlessUI which seems to be more active (last commit January, though).
